sound/soc/sof/ipc4-pcm.c

```diff
@@ -463,6 +452,38 @@ static int sof_ipc4_pcm_hw_free(struct snd_soc_component *component,
 	return sof_ipc4_trigger_pipelines(component, substream, SOF_IPC4_PIPE_RESET, 0);
 }
 
+static int sof_ipc4_pcm_pre_trigger(struct snd_soc_component *component,
+				    struct snd_pcm_substream *substream, int cmd)
+{
```

Comment:
I would have expected to see in the file first the pre, then the post, not the other way around.